### DISCUSSION TITLE
How to manually tell Symfony CLI where PHP.EXE is located on my Windows system?

### DISCUSSION THREAD

**Discussion by UweKeim** [symfony/symfony#47021] [Q&A] [answered]
On a Windows 11 system, I try to avoid installing a system-wide PHP version.

Therefore, when calling 

```cmd
symfony.exe" new --webapp my_app
```

I get the error message 

> no PHP binaries detected

What I want to do instead is to tell "symfony.exe" where the PHP installation is located (i.e. specify the folder/file path to "php.exe").

### My question

Is there any way to manually specify the location of "php.exe" for a call to "symfony.exe"?

Both, passing the path to "php.exe" in a command line argument or as some kind of configuration file would be OK.

--------

### Update 1

I'm aware that I can use something [like this](https://stackoverflow.com/a/67642155/107625):

```cmd
composer create-project symfony/website-skeleton your-project-name ^4.4
```

And I successfully managed to use this workaround. Still, using "symfony.exe" as the entry point would be my preferred way.

In my case I did something like:

```cmd
"c:\my_folder\php\php.exe" -f "c:\my_folder\comp\composer.phar" create-project symfony/skeleton "MyTestProject"
```

**Comment by Nek-** [comment]
Hello, did you try to add the PHP bin folder to your PATH?

**Reply by UweKeim** [reply]
Thanks. Actually, I do not want to modify my PATH or any other system variable. 

I want to be able to simply XCOPY everything to a system and have all ready to run, without modifications. Kind of a "portable" environment to check out from Git and run it right away.

If this is not possible with Symfony, no problem, I just want to be sure whether this is possible or not.

**Comment by stof** [comment, marked as answer]
I suggest opening a feature request on https://github.com/symfony-cli/symfony-cli

**Reply by UweKeim** [reply]
Merci beaucoup, thanks for your answers 😊.
